src/Components/LogExplorationPage.tsx

```diff
@@ -6,11 +6,11 @@ import { config } from '@grafana/runtime';
 import { SceneApp, useSceneApp } from '@grafana/scenes';
 
 import { initializeMetadataService } from '../services/metadata';
-import { makeIndexPage, makeRedirectPage } from './Pages';
+import { makeEmbedPage, makeIndexPage, makeRedirectPage } from './Pages';
```

Comment:
Nit.

```suggestion
import { makeEmbeddedPage, makeIndexPage, makeRedirectPage } from './Pages';
```

Reply:
Fixed in https://github.com/grafana/logs-drilldown/pull/1294/commits/93efb14e7ccfdab5e2e69d2f1e59d142f65fef7c